packages/connect/src/data/config.ts

```diff
@@ -197,12 +197,20 @@ export const config = {
                 'setBusy',
                 'unlockPath',
             ],
-            min: ['1.12.1', '2.5.3'],
+            min: { T1B1: '1.12.1', T2T1: '2.5.3' },
         },
         {
-            methods: ['showDeviceTutorial'],
-            min: ['0', '2.6.1'],
+            methods: ['showDeviceTutorial', 'authenticateDevice'],
+            min: { T1B1: '0', T2T1: '0', T2B1: '2.6.1' },
             comment: ['Only on T2B1'],
         },
+        {
+            methods: ['rebootToBootloader'],
+            min: { T1B1: '1.11.0', T2T1: '2.6.0' },
+        },
+        {
+            methods: ['getFirmwareHash'],
+            min: { T1B1: '1.11.1', T2T1: '2.5.1' },
+        },
```

Comment:
isn't authenticateDevice method missing in this list?

Reply:
it's added it different/previus commit:
https://github.com/trezor/trezor-suite/pull/9670/commits/6f40865ccea60fb593758a75f32c3e5eab088799

Reply:
aaa, i missed that